### PR TITLE
Setting up truffle

### PR DESCRIPTION
**Pull request by Janther**
moving things around and using truffle.

  - [x] Installing truffle 3.4.11
  - [x] setting up `package.json` to execute scripts properly
  - [x] moving contracts to contracts folder
  - [x] updating compiler version to `pragma solidity ^0.4.15;`
  - [x] removing `assert` function as it's now part of the language
  - [x] updating SafeMath to the lastest release
  - [ ] adapt shell scripts to work with the new folder structure
  - [ ] write migrations
  - [ ] write tests
  - [ ] write REAME

### CODE REVIEW

File: contracts/lib/SafeMath.sol [hunk 1]
```diff
@@ -0,0 +1,31 @@
+pragma solidity ^0.4.15;
```
Comment (matt-lough) [inline contracts/lib/SafeMath.sol:1]: Shouldn't we use the version controlled version of this using npm?

**Review by IsaacPayne — APPROVED** [review]
LGTM

**Review by matt-lough — CHANGES_REQUESTED** [review]
Use npm zeppelin-solidity for the safemath lib?

File: contracts/lib/SafeMath.sol (hunk 1 above)
Comment (Janther) [reply contracts/lib/SafeMath.sol:1]: good practice to compile everything with the same compiler

**Review by matt-lough — APPROVED** [review]

**Pull request merged**